spec/readme.md

```diff
@@ -139,6 +145,16 @@ Like the inputs, you can add JSON schema constraints as desired. Please specify
 
 The example has named one config option, called "speed", which must be an integer between zero and three, and another called "coordinates", which must be a set of three floats.
 
+### Contextual values
+
+Context inputs are values that are generally provided by the environment, rather than the human or process running the gear. These are generally values that are incidentally required rather than directly a part of the algorithm - for example, a license key.
+
+It is up to the gear executor to decide how (and if) context is provided. In the Flywheel system, the values can be provided by setting a `context.key-name` value on a container's metadata. For example, you could set `context.matlab_license_code: "AEX"` on the project, and then any gear running in that project with a context input called `matlab_license_code` would receive the value.
+
```

Comment:
Did we resolve how the full context will be assembled in the flywheel system? (i.e. do we walk up the tree, taking values in the closest container to populate the context?) Maybe this isn't the place to expound on details of the Flywheel system, but since Flywheel was mentioned, can this link to documentation that has more detail?

Reply:
Yes, and yes once it exists respectively. I do want to keep a clean separation - not much point to a spec if it can't be implemented at least twice - and the FW contexts will be implemented after this merge, so let's update with a link to docs.fw in a future PR.